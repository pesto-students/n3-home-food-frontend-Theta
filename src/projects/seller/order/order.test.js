import * as React from "react";
import Enzyme, { shallow } from "enzyme";
import EnzymeAdapter from "enzyme-adapter-react-16";
import SellerProducts from "./order";
import PastOrders from "./pastOrder";
import CurrentOrders from "./currentOrder";
Enzyme.configure({ adapter: new EnzymeAdapter() });

describe("Orders component", () => {
  let wrapper;
  test("render without error", () => {
    wrapper = shallow(<SellerProducts />);
    expect(wrapper.length).toBe(1);
  });

  test('should render correctly in "debug" mode', () => {
    const component = shallow(<SellerProducts />);

    expect(component).toMatchSnapshot();
  });
});

describe("Past Order Component", () => {
  let wrapper;
  let pastOrdersItem = []
  test("render without error", () => {
    wrapper = shallow(<PastOrders orders={pastOrdersItem} />);
    expect(wrapper.length).toBe(1);
  });

  test('should render correctly in "debug" mode', () => {
    const component = shallow(<PastOrders orders={pastOrdersItem} />);

    expect(component).toMatchSnapshot();
  });
});

describe("current Order Component", () => {
    let wrapper;
    let pastOrdersItem = []
    test("render without error", () => {
      wrapper = shallow(<CurrentOrders orders={pastOrdersItem} />);
      expect(wrapper.length).toBe(1);
    });
  
    test('should render correctly in "debug" mode', () => {
      const component = shallow(<CurrentOrders orders={pastOrdersItem} />);
  
      expect(component).toMatchSnapshot();
    });
  });
  
