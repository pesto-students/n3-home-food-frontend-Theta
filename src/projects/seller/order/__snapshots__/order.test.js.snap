// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Orders component Orders Component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;

exports[`Orders component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;

exports[`Past Order Component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;

exports[`current Order Component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;

exports[`testing seller graphs component Seller Graph Component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
